addons/mail/static/src/discuss/call/common/call_participant_card.js

```diff
@@ -129,6 +129,10 @@ export class CallParticipantCard extends Component {
         );
     }
 
+    get isActiveRtcSession() {
+        return this.rtcSession && this.rtcSession.eq(this.rtcSession.channel.activeRtcSession);
+    }
```

Comment:
Missing `channel?.activeRtcSession`, resulting to following crash:
<img width="1099" alt="Screenshot 2025-02-26 at 11 45 39" src="https://github.com/user-attachments/assets/1fb229a5-a5b2-48f8-b578-247dc1b860c2" />

Reply:
Do we already have a PR or task for the fix?

Reply:
Nope, just spotted the issue when posting the message